ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -631,6 +634,8 @@ objects:
           sed -n '/tasks/q;p' "${ASSETS_DIR}/network.yaml" > "${ASSETS_DIR}/down-containers.yaml"
           fi
 
+          BOOTSTRAP_FIP_IP="$(create_bootstrap_fip)
+
```

Comment:
don't need this. Just call the variable BOOTSTRAP_FIP_IP in the original create_bootstrap_fip function